extensions/commands/art/cmd_build_info.py

```diff
@@ -126,6 +126,14 @@ def __init__(self, graph, name, number, repository, build_url=None, with_depende
         self._cached_artifact_info = {}
         self._with_dependencies = with_dependencies
         self._add_cached_deps = add_cached_deps
+        self._conan_api = conan_api
+
+    def get_artifacts_folder(self, node, artifact_type):
+        if artifact_type == "package":
+            return node.get("package_folder")
+        else:
+            reference = RecipeReference.loads(node.get("ref"))
+            return self._conan_api.cache.export_path(reference)
```

Comment:
the package folder is ok in both conan create/install and export-pkg graphs, however for the re recipe_folder, in the export-pkg graph the path points to the user local folder, so here I resorted to query the conan cache to get the recipe path in the cache